Bottom: 4f3d293931fa91bad71e64e1774e528f75e4969c
Top:    b59ef3522bfc1e42b140680a2915de9abb0da877
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 14:13:31 +0000

Disable polygon heritage


---

diff --git a/src/polygon1.c b/src/polygon1.c
index a9ac914..5dfa616 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2827,8 +2827,10 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
-  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
-      return err_no_memory;
+//  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
+//      return err_no_memory;
+  a_copy = NULL;
+  b_copy = NULL;
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
